var/spack/repos/builtin/packages/zoltan/package.py

```diff
@@ -145,6 +147,16 @@ def configure_args(self):
             else:
                 config_args.append("--with-id-type=uint")
 
+        if "+scotch" in spec:
+            scotch_prefix = spec["scotch"].prefix
+            config_args.extend(
+                [
+                    "--with-scotch",
+                    "--with-scotch-incdir={0}".format(scotch_prefix.include),
+                    "--with-scotch-libdir={0}".format(scotch_prefix.lib),
+                ]
+            )
+
```

Comment:
```suggestion
        if spec.satisfies("+scotch"):
            scotch_prefix = spec["scotch"].prefix
            config_args.extend(
                [
                    "--with-scotch",
                    f"--with-scotch-incdir={scotch_prefix.include}",
                    f"--with-scotch-libdir={scotch_prefix.lib}",
                ]
            )

```

Two quick suggestions, otherwise this PR looks good to me. 
1) It's better to use `spec.satisfies()` instead of `if "+scotch" in spec` ([docs link](https://spack.readthedocs.io/en/latest/packaging_guide.html#testing-spec-constraints))
2) F-strings make it a bit easier to read compared to `.format()` calls.

Reply:
Thank you for the suggestions! The requested changes are done.